packages/helix-shared-config/src/schemas/index.schema.json

```diff
@@ -46,6 +46,22 @@
       "additionalProperties": {
         "$ref": "https://ns.adobe.com/helix/shared/query"
       }
+    },
+    "include": {
+      "type": "array",
+      "title": "Include",
+      "description": "Glob patterns for paths where this index is used.",
+      "items": {
+        "type": "string"
+      }
+    },
+    "exclude": {
+      "type": "array",
+      "title": "Exclude",
+      "description": "Glob patterns for paths where this index must not be used.",
+      "items": {
+        "type": "string"
+      }
```

Comment:
@trieloff can you take a quick look at this?

Reply:
Looks ok. You could add an array of `examples` to make it easier to understand. I don't think we have a regex for glob expressions.